### PR TITLE
feat: Update Chat Trigger node (no-changelog)

### PR DESCRIPTION
**Pull request by alexgrozav**
Address Chat trigger node review https://www.notion.so/n8n/Chat-trigger-review-ef6bdb355eec43aeb1dc5592f33c1d87

https://linear.app/n8n/issue/N8N-7029/chat-trigger-node

## Review / Merge checklist
- [x] PR title and summary are descriptive. **Remember, the title automatically goes into the changelog. Use `(no-changelog)` otherwise.** ([conventions](https://github.com/n8n-io/n8n/blob/master/.github/pull_request_title_conventions.md))
- [x] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [x] Tests included.
   > A bug is not considered fixed, unless a test is added to prevent it from happening again.
   > A feature is not complete without tests.

### CODE REVIEW

File: packages/@n8n/nodes-langchain/nodes/trigger/ChatTrigger/ChatTrigger.node.ts
Comment (StarfallProjects) [inline packages/@n8n/nodes-langchain/nodes/trigger/ChatTrigger/ChatTrigger.node.ts:1]: I think this is missing the `codex` section which includes the docs link and categories?

File: packages/@n8n/chat/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@n8n/chat",
-  "version": "0.4.1",
+  "version": "0.5.1",
```
Comment (netroy) [inline packages/@n8n/chat/package.json:3]: version for all packages are managed via the release process. we should be able to make manual releases with `-alpha` suffixes, but we should not be upgrading the versions in the repo.
But if that process doesn't work for this package, perhaps we should move this out into a separate repo?

Comment (alexgrozav) [reply packages/@n8n/chat/package.json:3]: I don't have a strong opinion about this. We're planning to replace the built-in chat with with components from the `@n8n/chat` package, and having it separately would probably make it a bit harder. I don't expect the components to change too often, though.

Comment (netroy) [reply packages/@n8n/chat/package.json]: btw, why are we publishing this package? is it not possible to use the local build of this in dev mode?

**Pull request merged**